### ISSUE TITLE
use scoped styles instead of global

### ISSUE THREAD

**Issue by zeratul1** [mazipan/vue2-simplert#40]
https://github.com/mazipan/vue2-simplert/blob/1bcf3cc58bc9a557b2ba9f5751511f4544e0f7e5/src/simplert.css#L1-L11

Hi,
Would you like to use scoped styles instead of these global ones.
I don't think it is a good practice that changing `box-sizing` of the root element in  simplert.

**Comment by mazipan** [comment]
can not use the `scope` style when inject html root element.
maybe you can hack the CSS for better implementation.
FYI : Pull Request are very welcome

**Comment by zeratul1** [comment]
Oh I'm sorry, 
I am not meaning the real `scope` style.
We could set this `box-sizing` style on simplert root instead of html root

```
.simplert { 
  box-sizing: border-box; 
} 
  
.simplert *, .simplert *:before, .simplert *:after { 
  box-sizing: inherit; 
}
```

**Comment by mazipan** [comment]
Haven't tried this one, do you have try those code @zeratul1 ?

**Comment by zeratul1** [comment]
Yes, I have tried these styles in my local package.
I'll make run the test and make a PR :D

**Comment by mazipan** [comment]
Close in https://github.com/mazipan/vue2-simplert/commit/48450d146c9651395d2a3ff84e901a991b93b0d3.
Published with version [0.5.5](https://github.com/mazipan/vue2-simplert/releases/tag/0.5.5)
